internal/pkg/releasenotes/release_notes.go

```diff
@@ -158,15 +161,15 @@ func (r *ReleaseNotes) ReadFromGithub() error {
 				break
 			}
 
-			// Search for PRs
-			issuesSearchResult, _, err := client.Search.Issues(ctx, commit.GetSHA(), nil)
+			pullRequests, _, err := client.PullRequests.ListPullRequestsWithCommit(ctx, owner, repo, commit.GetSHA(), nil)
 			if err != nil {
 				return err
 			}
 
-			if len(issuesSearchResult.Issues) > 0 {
-				body := issuesSearchResult.Issues[0].GetBody()
-				r.Merge(NewFromBody(body))
+			if len(pullRequests) > 0 {
+				releaseNotes := NewFromBody(pullRequests[0].GetBody())
+				log.Printf("%s: %v\n", commit.GetSHA(), releaseNotes)
```

Comment:
add a little description of what is being logged?

Reply:
```suggestion
				log.Printf("SHA: %s, Release Notes: %v\n", commit.GetSHA(), releaseNotes)
```